src/Text/index.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import * as React from "react";
+import * as tokens from "orbit-design-token";
+
+const colorText = {
+  primary: tokens.colorTextPrimary,
+  secondary: tokens.colorTextSecondary,
+  attention: tokens.colorTextAttention,
+};
+
+const weightText = {
+  regular: tokens.fontWeightNormal,
+  bold: tokens.fontWeightBold,
+};
+
+const sizeText = {
+  large: tokens.fontSizeTextLarge,
+  normal: tokens.fontSizeTextNormal,
+  small: tokens.fontSizeTextSmall,
+};
+
+type Props = {
+  type: $Keys<typeof colorText>,
+  size: $Keys<typeof sizeText>,
+  weight: "bold" | "regular",
```

Comment:
Why did you define above via `$Keys` and now you listed it all ?

please choose one way

Reply:
I forgot to change the weight prop via `$Keys`, I will fix this.